models/kernels/Pdm03.h

```diff
@@ -14,6 +14,7 @@ extern "C"
         double Cpar;
         double B;
         double XHuz;
+        double Huz;
```

Comment:
We really need to fix up all these variable names.  Will open an issue to do so.